vars/common.yml

```diff
@@ -10,19 +10,22 @@ router:
 api:
   routes:
     - dm-api-{env}.cloudapps.digital
+    - dm-api-{env}.cloudapps.digital/_metrics
   services:
     - digitalmarketplace_api_db
 
 search-api:
   routes:
     - dm-search-api-{env}.cloudapps.digital
+    - dm-search-api-{env}.cloudapps.digital/_metrics
   services:
     - search_api_elasticsearch
 
 antivirus-api:
   memory: 2G
   routes:
     - dm-antivirus-api-{env}.cloudapps.digital
+    - dm-antivirus-api-{env}.cloudapps.digital/_metrics
```

Comment:
Are we just doing the APIs first, or do we need this added to the other FEs too?

Reply:
Just apis for now